builtin/credential/cert/path_login.go

```diff
@@ -156,7 +156,12 @@ func (b *backend) verifyCredentials(req *logical.Request, d *framework.FieldData
 	clientCert := connState.PeerCertificates[0]
 
 	// Allow constraining the login request to a single CertEntry
-	certName := d.Get("name").(string)
+	var string certName
```

Comment:
Pull requests works fine, after changing the typo `var string certName` to `var certName string`. Thank!